guidelines/enhancement_template.md

```diff
@@ -265,6 +278,78 @@ enhancement:
 - Will any other components on the node change? For example, changes to CSI, CRI
   or CNI may require updating that component before the kubelet.
 
+### Impact of API Extensions
+
+Describe the API extensions here in detail, especially their impact on a cluster:
+ 
+- what are the SLIs (Service Level Indicators) an administrator or support can use to 
+- determine the health of the API extensions
```

Comment:
this could use a more explicit example for people to follow, e.g. "when api extension $foo was added, the following SLI was introduced...."

if i'm adding a new config resource for builds(for example), i am not clear what it would mean to add a new SLI for that resource, or if such a thing is even really applicable to that api change.